janusgraph-core/src/main/java/org/janusgraph/core/schema/JanusGraphManagement.java

```diff
@@ -410,34 +410,34 @@
      *
      * @return a collection of all the mgmt API schema printing methods
      */
-    public String printSchema();
+    String printSchema();
```

Comment:
Doesn't this change make these methods package-private?

Reply:
No, this file is an interface. https://stackoverflow.com/a/161787

Reply:
As said by @farodin91 . All methods in the interface are `public`. This is redundant